src/System.Text.Json/src/System/Text/Json/JsonDocument.CustomDb.cs

```diff
@@ -0,0 +1,238 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace System.Text.Json
+{
+    public sealed partial class JsonDocument
+    {
+        // The database for the parsed structure of a JSON document.
```

Comment:
can you make this triple-slash comments in a <summary> block to be consistent with existing source files under the System.Text.Json directory?

Reply:
github stripped off my 'summary' tag from the comment above

Reply:
I could... but this is a private nested type; I don't think it makes sense for having an intellisense experience (and it's not a seed to the doc writing).  (And, of course, this is much, much bigger than a "summary" :smile:)